binder/src/main/java/io/grpc/binder/InternalPeerUid.java

```diff
@@ -0,0 +1,21 @@
+package io.grpc.binder;
```

Comment:
I remember discussing the need for this class in the past, but now I'm wondering if we still need it.

(If I've missed something, please ignore, but if not I suspect we can leave this out).

Reply:
I think you're right - it was previously needed when a change was made to internal/BinderTransportSecurity to populate the PeerUid unconditionally, since that class is in a different package from the other PeerUid classes. But now this is all handled by the interceptor in the same package, so we have no need to access the package-private methods in PeerUid(s) outside the package itself. Removing.